src/bin/cernan.rs

```diff
@@ -189,7 +189,7 @@ fn main() {
     // BACKGROUND
     //
 
-    let flush_interval = args.flush_interval;
+    let flush_interval = 1; // TODO: use constant
```

Comment:
TODO left. That said, the optimizer should detect that this is a constant so I'm OK with not introducing some global constant.